subprojects/core-api/src/main/java/org/gradle/api/artifacts/result/ResolvedComponentResult.java

```diff
@@ -38,23 +38,32 @@
      * <p>The return type is declared as an opaque {@link ComponentIdentifier}, however the identifier may also implement one of the following interfaces:</p>
      *
      * <ul>
-     *     <li>{@link org.gradle.api.artifacts.component.ProjectComponentIdentifier} for those component instances which are produced by the current build.</li>
-     *     <li>{@link org.gradle.api.artifacts.component.ModuleComponentIdentifier} for those component instances which are found in some repository.</li>
+     * <li>{@link org.gradle.api.artifacts.component.ProjectComponentIdentifier} for those component instances which are produced by the current build.</li>
```

Comment:
You probably didn't intend to change the indent (pun intended, oh oh!)

Reply:
:). Fixed